Scalar/CommandLine/ScalarVerb.cs

```diff
@@ -413,6 +413,24 @@ protected void ValidatePathParameter(string path)
             }
         }
 
+        protected void ValidateUrlParameter(string url)
+        {
+            if (url.StartsWith("http://") ||
+                url.StartsWith("https://") ||
+                url.StartsWith("ssh://"))
+            {
+                return;
+            }
+
+            // Do not allow other protocols or absolute file paths.
+            if (url.Contains("://") || url.StartsWith('/'))
+            {
+                this.ReportErrorAndExit($"Invalid URL: '{url}'");
```

Comment:
maybe make the error message say something to the effect that only http, https, and ssh urls are supported.